Library/AppEnvironment.swift

```diff
@@ -82,6 +82,8 @@ public struct AppEnvironment: AppEnvironmentType {
     let storage = AppEnvironment.current.cookieStorage
     storage.cookies?.forEach(storage.deleteCookie)
 
+    // Resetting the segment client
+    AppEnvironment.current.ksrAnalytics.identify(newUser: nil)
```

Comment:
@singhhari have we done this previously? Curious about what's the outcome of this call 🤔

Reply:
Yeah, we were doing this before but it was baked into the behavior of the property observer on `loggedInUser`. If loggedInUser was not set we were passing `nil` into the `identify` method and that causes the client to reset the traits and user id.

So we're still doing this now but because I removed the property observer I now have to explicitly reset it by passing in nil on log out.